.vscode/settings.json

```diff
@@ -0,0 +1,11 @@
+{
+    "gopls": {
+        "build.buildFlags": [
+            // Context: https://github.com/ava-labs/avalanchego/pull/3173
```

Comment:
I'm surprised to see that this works, but wouldn't touch it if it does.

Reply:
Is is documented somewhere that this is supported? This also surprises me.

Reply:
It is documented: https://code.visualstudio.com/docs/languages/json#_json-with-comments. TIL